src/Admin/index.tsx

```diff
@@ -1,75 +1,73 @@
+import React from "react";
 import { useParams, Outlet } from "react-router-dom";
 
 export const Admin = () => {
```

Comment:
Have we agreed on 2 spaces indentation? I don't remember having that conversation. I personally prefer 4.

Reply:
Lets discuss in devcoffee tomorrow. I was initially of the same opinion but I've seen a light.